src/components/search/detailed/AppSearchResults.js

```diff
@@ -139,7 +139,7 @@ export default function AppSearchResults(props) {
                     )[1];
                     const { t } = useTranslation("common");
                     return (
-                        <Grid spacing={1}>
+                        <Grid container spacing={1}>
```

Comment:
This fixed a warning that `spacing` can only be used on a `container` (which this is).